reloader/reloader.py

```diff
@@ -20,21 +20,50 @@ def dprint(*args, fill=None, fill_width=60, **kwargs):
     print("[Package Reloader]", *args, **kwargs)
 
 
+def path_contains(a, b):
+    return a == b or b.startswith(a + os.sep)
```

Comment:
it may not work for symlinks. You may need to resolve a and b first by `os.path.realpath`

Reply:
I think that this should be the correct symlink behavior. I have `User` symlinked to a Dropbox directory, but `sys.modules['User'].__path__` is `_NamespacePath(['/Users/thom/Library/Application Support/Sublime Text 3/Packages/User'])
` and `sys.modules['User.dump_highlighting_command'].__file__` is `'/Users/thom/Library/Application Support/Sublime Text 3/Packages/User/dump_highlighting_command.py'`. So as long as we *don't* call `os.path.realpath`, I think things should work out the right way. If we do call it, then we could have a problem if there is a module outside the physical package directory symlinked from within it.

This might not apply if the `Packages` directory itself is symlinked. I'll check tomorrow; it's easier to set up test installs on Linux.

Reply:
I've checked on a portable Linux install with `Packages` and `Installed Packages` symlinked elsewhere, and it seems to work as expected. The `__path__` and `__file__` attributes of package modules are all under `sublime.packages_path()` or `sublime.installed_packages_path()` even though their "real" locations aren't.

Generally, I think we should be fine as long as we consistently work with logical paths rather than calling `os.path.realpath`.

Reply:
@randy3k Just pinging in case you were waiting on me.

Reply:
Sorry for the late reply. I am very busy recently and had no time to review any PRs.
I would want to test it on my own machine before merging it. It will happen no later than the Christmas.